components/Encode/components/EncodeSetData.tsx

```diff
@@ -124,8 +124,8 @@ const EncodeSetData: React.FC<Props> = ({ web3, isBatch }) => {
     <>
       {createInputs(keyValuePairs)}
       {isBatch ? (
-        <div className="columns">
-          <div className="column">
+        <div>
+          <div className="my-4 mr-4">
```

Comment:
I'm not certain this mr-4 is responsive. I preferred the previous way with columns.

Reply:
Columns class is overwriting the left offset of the element of the container. Thats why i removed it. All margin classes are responsive (including `mr`) as they use `rem`